package/MDAnalysis/topology/guessers.py

```diff
@@ -0,0 +1,537 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+"""
+Guessing unknown Topology information --- :mod:`MDAnalysis.topology.guessers`
+=============================================================================
+
```

Comment:
Add a textual deprecation, too:
```suggestion

.. deprecated:: 2.8.0
   The :mod:`MDAnalysis.topology.guessers` module will be removed in release 3.0.0.
   It is deprecated in favor of the new Guessers API. See 
   :mod:`MDAnalysis.guesser.default_guesser` for more details.
   
```

Reply:
If you merge the suggestion then you can also check "docs" box ;-)

Reply:
(Not super-important because theses docs are not rendered anywhere so really treat it as a suggestion.)